src/mlpack/methods/ann/layer/linear_impl.hpp

```diff
@@ -41,6 +41,62 @@ Linear<InputDataType, OutputDataType, RegularizerType>::Linear(
   weights.set_size(outSize * inSize + outSize, 1);
 }
 
+template<typename InputDataType, typename OutputDataType,
+    typename RegularizerType>
+Linear<InputDataType, OutputDataType, RegularizerType>::Linear(
+	const Linear& layer) :
+    inSize(layer.inSize),
+    outSize(layer.outSize),
+    weights(layer.weights),
+    regularizer(layer.regularizer)
+{
+  // Nothing to do here.
+}
+
+template<typename InputDataType, typename OutputDataType,
+    typename RegularizerType>
+Linear<InputDataType, OutputDataType, RegularizerType>::Linear(
+  Linear&& layer) :
```

Comment:
```suggestion
    Linear&& layer) :
```
Another style fix. :)